myRobot/src/main/java/MyRobot.java

```diff
@@ -5,50 +5,64 @@
 /* the project.                                                               */
```

Comment:
I know you're using this for local testing, but this should be removed for the PR.  Something similar to this should be put into the examples.